client/app/styles/app.scss

```diff
@@ -20,66 +18,31 @@ $tab-content-border: $silver;
 $tab-content-color: inherit;
 $tab-content-padding: 1rem 0 0;
 $fieldset-border: $callout-border;
-// $fieldset-padding: rem-calc(20);
-// $fieldset-margin: rem-calc(18 0);
-
-
-//
-// Foundation
-// --------------------------------------------------
-@import 'foundation-sites/foundation';
-
-// Override Foundation's foundation-print-styles mixin
-// @import 'foundation-print';
 
+@import 'foundation';
```

Comment:
What is this for ? I thought we're moving away from foundation?

Reply:
@godfreyyeung no not yet, I'm still preserving everything foundation-related because I'm not ready to overhaul the entire thing yet. This is a seemingly "newer" way to import foundation dependencies.